### PR TITLE
Add TTS finetuning interface

### PR DESCRIPTION
**Pull request by Spycsh**
## Type of Change

Feature
## Description

I add the TTS finetuning code

## Expected Behavior & Potential Risk

 TTS finetuning interface is provided

## How has this PR been tested?

None

## Dependency Change?

None

### CODE REVIEW

File: intel_extension_for_transformers/neural_chat/examples/tts_finetuning/inference_peppa.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    return " ".join(results)
+
+
+while True:
+    try:
+        text = input("Write a sentence to let peppa pig speak:\n")
+        text = correct_abbreviation(text)
+        text = correct_number(text)
+        inputs = processor(text=text, return_tensors="pt")
+        spectrogram = model.generate_speech(inputs["input_ids"].cuda(), speaker_embeddings.cuda())
```
Comment (lvliang-intel) [inline intel_extension_for_transformers/neural_chat/examples/tts_finetuning/inference_peppa.py:126]: why only on cuda?

File: intel_extension_for_transformers/neural_chat/examples/tts_finetuning/inference_peppa.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        speaker_embeddings = torch.nn.functional.normalize(speaker_embeddings, dim=2)
+        speaker_embeddings = speaker_embeddings.squeeze().cpu().numpy()
+    return speaker_embeddings
+
+audio_dataset = Dataset.from_dict({"audio": [os.path.join(workdir, "audios/4.mp3")]}).cast_column("audio", Audio(sampling_rate=16000))
+sembeddings = create_speaker_embedding(audio_dataset[0]["audio"]['array'])
+speaker_embeddings = torch.tensor(sembeddings).unsqueeze(0)
+vocoder = SpeechT5HifiGan.from_pretrained("microsoft/speecht5_hifigan").cuda()
+
+def correct_abbreviation(text):
```
Comment (lvliang-intel) [inline intel_extension_for_transformers/neural_chat/examples/tts_finetuning/inference_peppa.py:54]: Can we use the neural_chat tts code to do inference?

**Comment by kevinintel** [comment]
we never release mp3 file before, I will check if it need additional review or not

**Comment by hshen14** [comment]
> we never release mp3 file before, I will check if it need additional review or not

we can use document to help users download the mp3 and generate the text. Liang, we need such a tool under Neuralchat.

File: intel_extension_for_transformers/neural_chat/examples/tts_finetuning/inference_peppa.py (hunk 1 above)
Comment (Spycsh) [reply intel_extension_for_transformers/neural_chat/examples/tts_finetuning/inference_peppa.py]: CPU training and inference added.

File: intel_extension_for_transformers/neural_chat/examples/tts_finetuning/inference_peppa.py (hunk 2 above)
Comment (Spycsh) [reply intel_extension_for_transformers/neural_chat/examples/tts_finetuning/inference_peppa.py:54]: Will cause _pickle.UnpicklingError if I want to refer to that code

**Comment by Spycsh** [comment]
This PR will only add TTS finetuning interface.
A more robust example will be added later considering the copyrights and license issues.

**Review by hshen14 — APPROVED** [review]

**Comment by hshen14** [comment]
@lvliang-intel please review.

**Pull request merged**